en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-2-all-in-one-gw.md

```
@@ -559,27 +663,36 @@ helm install <release-name> <helm-chart-path> \
 kubectl get ing -n <NAMESPACE>
 ```
 
+Use the value from `ADDRESS` or `EXTERNAL-IP` as the external IP.
+
 If the defined hostnames (in the previous step) are backed by a DNS service, add a DNS record mapping the hostnames and
 the external IP (`EXTERNAL-IP`) in the relevant DNS service.
 
 If the defined hostnames are not backed by a DNS service, for evaluation purposes you may add an entry mapping the
 hostnames and the external IP in the `/etc/hosts` file on the client side:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)